### ISSUE TITLE
Site Improve error

### ISSUE THREAD

**Issue by jamesjacobs** [jadu/selectWoo#5]
The Site Improve chrome extension reports a miss-use of aria error. The error relates to the lack of `aria-controls` on the `span` with the `role="combobox"` attribute. In this case, the `combobox` should have the `aria-controls` attribute with a value equal to the ID of the dropdown. However, the dropdown isn't present in the DOM until the dropdown is opened.

Potential fix - add `aria-controls` with a value equal to the ID of the dropdown container (`.dropdown-wrapper`). This would require the dropdown to be added to the dropdown container, rather than elsewhere in the DOM and a unique ID adding to the wrapper.

Needs more investigation and testing both in the repo playground and in product.

![Screenshot 2020-01-23 at 17 01 22](https://user-images.githubusercontent.com/756393/73006121-02706b00-3e02-11ea-83df-bab588cd4a2d.png)

**Comment by markbjadu** [comment]
![Screenshot 2020-07-08 at 13 41 09](https://user-images.githubusercontent.com/6906173/86932610-7fce3480-c131-11ea-952c-ad1f7683ff13.png)

![Screenshot 2020-07-08 at 15 43 12](https://user-images.githubusercontent.com/6906173/86932847-c6239380-c131-11ea-9cf8-2144ce125e28.png)

In the XFP received forms list it's flagging them. I think these are related to filter bar search? Might be others as well.
